testcafe/tests/pages/photos-album-model.js

```diff
@@ -0,0 +1,273 @@
+import { t, Selector } from 'testcafe'
+import {
+  getPageUrl,
+  getElementWithTestId,
+  isExistingAndVisibile
+} from '../helpers/utils'
+import PhotoPage from '../pages/photos-model'
+const photoPage = new PhotoPage()
+
+export default class Page {
+  constructor() {
+    this.albumEmptyText = photoPage.folderEmpty
+      .child('h2')
```

Comment:
Pourquoi pas de data-test-id sur ce h2 ?

Reply:
Parce que le composant Empty sur lequel je me base pour définir photoPage.folderEmpty est :
```
  <Empty
      data-test-id="empty-folder"
      icon={EmptyIcon[type]}
      title={localeKey ? t(`empty.${localeKey}_title`) : t('empty.title')}
      text={
        (localeKey && t(`empty.${localeKey}_text`)) ||
        (canUpload && t('empty.text'))
      }
    />
```
Donc pas de possibilité de modifier le h2 directement, et vérifier le texte `You don't have any album yet` fait partie des étapes demandées dans le scenario
(je ne peux donc vérifier juste l'existance de photoPage.folderEmpty)